pkg/controller/persistentvolume/persistentvolume_host.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolume
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/cloudprovider"
+	"k8s.io/kubernetes/pkg/types"
+	"k8s.io/kubernetes/pkg/util/io"
+	"k8s.io/kubernetes/pkg/util/mount"
+	vol "k8s.io/kubernetes/pkg/volume"
+)
+
+// VolumeHost interface implementation for PersistentVolumeController.
```

Comment:
Agreed. This is an unfortunate requirement of the current factoring of volume plugin initialization code. It requires kubelet specific methods used by the mounting code to be implemented by all initializers even if the initializer does not do mounting (like the controllers). All of the volume plugin initialization code (how the plugins are initialized, factoring and initialization of context, etc.) all need a good redesign. But that's outside the scope of this work. 
I have https://github.com/kubernetes/kubernetes/issues/14217#issuecomment-186470305 opened where we can address this.